### ISSUE TITLE
Split git-issue.sh into multiple files

### ISSUE THREAD

**Issue by vyrondrosos** [dspinellis/git-issue#55]
With the main script file nearing 1200 lines of code --and my [GSoC version](https://github.com/eellak/gsoc2019-git-issue/blob/new_attributes/git-issue.sh) at 1500-- and getting a bit unwieldy, I was considering splitting out some of the functions in a secondary source file(for example the ones concerning _GitHub/GitLab_ `import/export`). Would that be OK, or would it be preferable to keep it to the single file, e.g for easier installation?

**Comment by dspinellis** [comment]
Good point!  Some files would then need to be installed in a library folder (e.g. `/usr/lib/git-issue`.  Is there precedent for this?  Where are Git internal helper files located?  How is then installation handled?   Alternatively, the split could involve creating some internal helper binaries e.g. `git-issue-import` `git-issue-export` and install them as usual in a `bin` folder.

**Comment by vyrondrosos** [comment]
Those are some good points! It seems that git does install some library script files under `/usr/lib/git-core`, so I think that installing a `/usr/lib/git-issue` directory wouldn't be unprecedented. The 'separate executables for some subcommands' method is certainly interesting and could be useful but I believe there still would need to be a library file for them to source, since most commands currently rely on some common helper functions.
